src/Youtube.js

```diff
@@ -33,6 +33,10 @@ THE SOFTWARE. */
 }(this, function(videojs) {
   'use strict';
 
+  if (!videojs.getComponent) {
```

Comment:
We actually should be doing that considering there's a note in the README about using videojs 5 with the tech, going to remove this check.